src/Hello.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Hello = function(props) {
```

Comment:
You can absolutely use the ES6 arrow functions here. Another alternative would be, to not use const, but a named function. That is sometimes better picket up by code inspection tools in IDEs. 

So those approaches would both work: 
```
const Hello = props => {

};

// Alternative, more verbose without having to declare const:
function Hello(props) {
  // ...
}
```
Above example is fine too, though.